src/server/routes/eval.ts

```diff
@@ -168,9 +170,12 @@ evalRouter.patch('/:id/author', async (req: Request, res: Response): Promise<voi
 
 evalRouter.get('/:id/table', async (req: Request, res: Response): Promise<void> => {
   const { id } = req.params;
-  const limit = Number(req.query.limit) || 50;
-  const offset = Number(req.query.offset) || 0;
-  const filterMode = EvalResultsFilterMode.parse(req.query.filterMode) ?? 'all';
+  const format = req.query.format as string | undefined;
```

Comment:
Use an enum: a value other than `csv` or `json` will result in unpredictable behaviors due effects on limit and offset.